util/godot/classes/rigid_body_3d.h

```diff
@@ -2,7 +2,14 @@
 #define ZN_GODOT_RIGID_BODY_3D_H
 
 #if defined(ZN_GODOT)
-#include <scene/3d/physics_body_3d.h>
+#include <core/version.h>
+
+#if VERSION_MAJOR == 4 && VERSION_MINOR <= 2
+#include <scene/3d/rigid_body_3d.h>
```

Comment:
Should be `#include <scene/3d/physics_body_3d.h>`

Reply:
Sorry, missed that one